dist/docsite/css/cdr-tokens.css

```diff
@@ -1,5 +1,5 @@
 :root {
-  --cdr-text-size-root: 10px;
+  --cdr-text-size-root: 1rem;
```

Comment:
I think this needs to remain as 10px or we will be saying the root size is 16px correct?